### ISSUE TITLE
Change Miner Exe Name After Unpack

### ISSUE THREAD

**Issue by DARK-DEVIL-66** [UnamSanctam/SilentCryptoMiner#1914]
Today i checked My personal computer Miner name is svschost but after run it's unpacking like random name like yegfyuef.exe/ewwuef.exe it's easy to user can kill/block if you have any solution for next update

**Comment by DARK-DEVIL-66** [comment]
and i enabled root kit future but it's not working properly ?miner are visible

**Comment by UnamSanctam** [comment]
You choose the filename name inside the "Startup" tab. And if you have the rootkit enabled then the miner and watchdog will be injected into `dialer.exe`, so if you can't see any `dialer.exe` then the miner is hidden.

**Comment by DARK-DEVIL-66** [comment]
it's defult chrome.exe under programe folder in Startup section i can't edit only on off are work bro

**Comment by DARK-DEVIL-66** [comment]
why miner status are  Stopped

**Comment by UnamSanctam** [comment]
If it's called something like yegfyuef.exe/ewwuef.exe and it's not that in the "Startup" tab then you're using some sort of crypter or something that does that. The miner installs itself using the "File Name" inside the "Startup" tab.

>why miner status are Stopped

Either it cannot connect to the pool or the GPU isn't working correctly (if it's a GPU miner).

**Comment by DARK-DEVIL-66** [comment]
No Cpu And Sometime it's show active but hashrate are 0

**Comment by DARK-DEVIL-66** [comment]
![Screenshot 2023-11-12 at 23-02-53 Unam Web Panel Unam Sanctam(1)](https://github.com/UnamSanctam/SilentCryptoMiner/assets/149578298/52787a2e-1125-4708-8f33-82feef61d75c)
![Screenshot 2023-11-12 at 23-03-19 Unam Web Panel Unam Sanctam](https://github.com/UnamSanctam/SilentCryptoMiner/assets/149578298/2c3769c4-4360-47ad-9ce0-44a073c4821c)
![Screenshot 2023-11-12 at 23-03-46 Unam Web Panel Unam Sanctam](https://github.com/UnamSanctam/SilentCryptoMiner/assets/149578298/f560d12f-6223-4b06-b443-ae12bead61cd)

**Comment by UnamSanctam** [comment]
For the CPU miner "Stopped" only happens if the pool connection isn't working. Hashrate at 0 almost always means that there isn't enough resources to mine (usually memory/RAM) or something is stopping it from using the cores. That miner is running on a virtual machine, meaning that it most likely is limited on how much RAM and CPU cores it can use.

**Comment by DARK-DEVIL-66** [comment]
Intel(R) Core(TM) i7-8750H CPU @ 5.00 GHz
 Intel(R) Xeon(R) Gold 6130 CPU @ 2.10GHz
AMD EPYC 7763 64-Core Processor 
4Gb/8GB Ram Running on real PC

**Comment by UnamSanctam** [comment]
Can you show a screenshot? Since the one in your previous screenshot isn't running on a real PC.

**Comment by DARK-DEVIL-66** [comment]
Yes it's Running real PC

**Comment by DARK-DEVIL-66** [comment]
![Screenshot 2023-11-12 at 23-17-43 Unam Web Panel Unam Sanctam](https://github.com/UnamSanctam/SilentCryptoMiner/assets/149578298/badfc666-868e-4dbd-bc2f-aac5130e48c2)

**Comment by DARK-DEVIL-66** [comment]
i think it's error happen in configration

**Comment by DARK-DEVIL-66** [comment]
{
    "algo": "rx/0",
    "pool": "xmr.2miners.com",
    "port": 12222,
    "wallet": "4B9ttGq1dRJgqmnGJ6HLeZ7VhdTxvQS67Nb4FiN5CXKNNy6mJnYvTSzdyXxrfUF1mcEx4VzqzA4mBRSDxCf6aCyL1D2hSnC.{COMPUTERNAME}",
    "password": "x",
    "nicehash": false,
    "ssltls": true,
    "max-cpu": 50,
    "idle-wait": 5,
    "idle-cpu": 80,
    "stealth-targets": "Taskmgr.exe,ProcessHacker.exe,perfmon.exe,procexp.exe,procexp64.exe",
    "kill-targets": "",
    "stealth-fullscreen": false
}


it's my config for xmr

**Comment by DARK-DEVIL-66** [comment]
{
    "url": "stratums://RA4su9dVWToKfNiygSGw3uwHvBSDC7B45m.{COMPUTERNAME}@rvn.2miners.com:16060",
    "algo": "kawpow",
    "max-gpu": 50,
    "idle-wait": 1,
    "idle-gpu": 90,
    "stealth-targets": "Taskmgr.exe,ProcessHacker.exe,perfmon.exe,procexp.exe,procexp64.exe",
    "kill-targets": "",
    "stealth-fullscreen": false
}

**Comment by DARK-DEVIL-66** [comment]
> Can you show a screenshot? Since the one in your previous screenshot isn't running on a real PC.

it's running on my clients pc so i cannot get realtime pc screen shots

**Comment by UnamSanctam** [comment]
>Yes it's Running real PC

No the EPYC one isn't, you can see the `Microsoft Hyper-V Video` listed as the GPU.

>i think it's error happen in configration

Then it shouldn't say "Active" in 99% of cases. And your configuration is correct. It's almost always due to a lack of available CPU cores (or cache) or RAM, you can temporarily switch that miner to another configuration that isn't mining RandomX since it's the most demanding of those algorithms.

**Comment by DARK-DEVIL-66** [comment]
u mean try to new pool/algo (GhostRider or CryptoNight) ?

**Comment by UnamSanctam** [comment]
>u mean try to new pool/algo (GhostRider or CryptoNight) ?

Yes on those machines with 0H/s then you can switch only their configuration to a different configuration that uses another less resource demanding algorithm, doing that makes it easier to see if the reason for 0H/s with RandomX is due to a lack of resources.

**Comment by DARK-DEVIL-66** [comment]
please send me configration

**Comment by DARK-DEVIL-66** [comment]
whats happen running 2miners differnd config in same pc

**Comment by UnamSanctam** [comment]
>please send me configration

It depends on what you want to test mine:
```
{
"algo": "gr",
"pool": "eu.flockpool.com",
"port": 5555,
"wallet": "RAPTOREUMWALLET.{COMPUTERNAME}",
"ssltls": true,
"max-cpu": 50,
"idle-wait": 5,
"idle-cpu": 80,
"stealth-targets": "Taskmgr.exe,ProcessHacker.exe,perfmon.exe,procexp.exe,procexp64.exe",
"kill-targets": "",
"stealth-fullscreen": false
}
```

>whats happen running 2miners differnd config in same pc

Not sure what you're asking. But if you change the configuration then it will change it.

**Comment by DARK-DEVIL-66** [comment]
i using rvn coin  so i pasted config detiles in Default Etherium Tab it's Correct or wrong ?

**Comment by DARK-DEVIL-66** [comment]
![Screenshot 2023-11-13 at 01-44-50 Unam Web Panel Unam Sanctam](https://github.com/UnamSanctam/SilentCryptoMiner/assets/149578298/16df00a3-1e88-40e6-81df-1ee28b50fdd3)

**Comment by DARK-DEVIL-66** [comment]
or ineed to create seperate Default Raven Coin tab

**Comment by DARK-DEVIL-66** [comment]
![Screenshot 2023-11-13 at 12-17-01 Miner Statistics - Best Monero XMR Mining Pool - 2Miners](https://github.com/UnamSanctam/SilentCryptoMiner/assets/149578298/527f0369-eab3-4db0-8543-4bf232c43417)


how is'it possible and what you think
